_posts/0000-01-02-Tgodna19.md/Tahira

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
Just starting out on this thing called coding! Excited to see where it will take me!